### ISSUE TITLE
Not recognised as an audio device

### ISSUE THREAD

**Issue by FilipSzkandera** [Binary-Eater/Pi-Bluetooth-Audio-Visualizer#1]
Hi,

I've went step-by-step by your readme on a clean RPi 64-bit lite OS, but `python-gobject` and `python-gobject-2` could not be installed, and even when all of other things were done (including the bluetooth pairing), my device could pair successfully, but it could not stream any music to it.

**Comment by Binary-Eater** [comment]
It's been a while since I have used this setup. When you pair your device and then connect it, does the Raspberry Pi show up as a media device through the device you connected with?

**Comment by FilipSzkandera** [comment]
Nope, I can only connect to it. I've tried different softwares before and I verified I could play music through bluetooth, so maybe there is a step I missed during the setup...

**Comment by Binary-Eater** [comment]
Did you update `/etc/bluetooth/audio.conf` and start pulseaudio before connecting your device to the Pi? I think if you don't, that would explain why the Pi does not show up as an audio device over Bluetooth.

**Comment by Binary-Eater** [comment]
If running `pulseaudio -D` before the next time you connect your device to the Pi doesn't work, I can look at verifying this on a new Raspbian Lite image to see if updates need to be made to the instructions.
